documentation/how-to/index.en.md

```diff
@@ -5,20 +5,14 @@ tx_slug: documentation_how-to_index
 
 # How-to guides
 
-QField leverages the power of QGIS to comfortably configure your
-projects. After setting up everything, you can eventually synchronise your
-project and any additional files to your device and immediately get
-down to work.
+QField leverages the power of QGIS to comfortably configure your projects.
```

Comment:
This is a bit of a weird sentence. QField leverages the power of QGIS to deliver an amazing field experience, it isn't used to configure projects :) 

I think what it meant to suggest here is that QField being built on top of QGIS allows users configure projects within their favorite GIS application, namely QGIS? :)